movies-app/src/views/DetailsPage.jsx

```diff
@@ -18,21 +19,26 @@ const DetailsPage = function () {
   return (
     <>
       {data && (
-        <Stack direction="row">
+        <Stack direction={isMobile ? 'column' : 'row'} justifyContent="center">
           <img
-            className="width-300"
+            style={{ width: isMobile ? '80vw' : '300px' }}
```

Comment:
Just for future reference: https://itnext.io/writing-media-queries-with-sass-mixins-3ea591ea3ea4